### PR TITLE
BA-670: Adjust quote markdown design

### PR DESCRIPTION
**Pull request by JacobMu**
[JIRA task](https://inovaciebratislava.atlassian.net/browse/BA-670)

image

### CODE REVIEW

File: next/components/atoms/HomepageMarkdown.tsx [hunk 1]
```diff
@@ -144,7 +144,7 @@ export const HomepageMarkdown = ({ className, content, numericalList }: Homepage
             <div className="flex gap-x-8 lg:gap-x-6">
               <div
                 className={cx(
-                  'h-4 w-4 shrink-0 bg-category-600 rounded-full mt-1 border-4 border-solid border-category-600',
+                  'h-4 w-4 shrink-0 bg-category-600 rounded-full mt-2 border-4 border-solid border-category-600',
```
Comment (mpinter) [inline next/components/atoms/HomepageMarkdown.tsx:147]: should this change be here ? does not seem related, and if it is it also changes other lists which is not wanted

**Review by mpinter — COMMENTED** [review]
check / remove the change for `li` component (or let me know fi I misunderstood), otherwise looks fine

File: next/components/atoms/HomepageMarkdown.tsx (hunk 1 above)
Comment (JacobMu) [reply next/components/atoms/HomepageMarkdown.tsx:147]: This change was to adjust a bullet point more to center with the first line. But now when I look into it, the previous value was good enough.

**Review by mpinter — APPROVED** [review]
